lib/bundlex/cnode/supervisor.ex

```diff
@@ -0,0 +1,92 @@
+defmodule Bundlex.CNode.Supervisor do
```

Comment:
Consider renaming this module, this may be confused with supervisors known from OTP. Your suggestion was `CNode.Server` + renaming NameServer -> NameStore

Reply:
Looking again at the code of CNode I think it's perfectly fine to merge them as it is a common pattern to have GenServer module with callbacks and public API for using it, but I won't insist, do as you wish